src/ripple/overlay/impl/OverlayImpl.cpp

```diff
@@ -829,7 +829,7 @@ OverlayImpl::getOverlayInfo()
             auto version{sp->getVersion()};
             if (!version.empty())
                 // Could move here if Json::value supported moving from strings
-                pv[jss::version] = version;
+                pv[jss::version] = std::string{version};
```

Comment:
Good place here for that `operator= (std::string_view const&)` overload I mentioned.

Reply:
hmm okay